projects/crypto-currency/destemidos/App/src/components/Home/CoinsDetails.jsx

```diff
@@ -1,26 +1,90 @@
-import React from 'react';
-import { ContainerDiv, ButtonDiv, StyledButton } from './Style.js'
+import React from 'react'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import CompareArrowsIcon from '@material-ui/icons/CompareArrows'
 import { Link } from 'react-router-dom'
 import './Style.css'
-import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart'
-import CompareArrowsIcon from '@material-ui/icons/CompareArrows'
 
-function CoinsDetails() {
-  return (
-    <ContainerDiv>
-      <h4>Coins</h4>
-      <div>0,09 BTC</div>      
-      <ButtonDiv>
-        <Link className="LinkButton" to='/mercado'>
-          <StyledButton variant="contained" color="primary">
-          Comprar<AddShoppingCartIcon/>
-        </StyledButton></Link>
-        <StyledButton variant="contained" color="secondary">
-          Vender<CompareArrowsIcon/>
-        </StyledButton>
-      </ButtonDiv>    
-    </ContainerDiv>
-  )
+//linhas 10 até 61 vão ser retiradas, foram colocadas apenas a fim de visualização do layout
+function createData(name, calories, fat, carbs, protein) {
+  return { name, calories, fat, carbs, protein };
 }
```

Comment:
só confirmando, aqui será substituído pelo conteúdo do back-end correto?

Reply:
Lembrar de tirar o ";" quando for fazer a substituição, ali na linha 10

Reply:
tirei, isso mesmo será substituido pelo conteudo do back end